### ISSUE TITLE
No single pointer provided to drag the "columns headers" to the required position in the "Polyglot Notebook" grid table: A11y_Testing, Interactive and ARM tools_Table_Screen Reader.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/interactive#3597]
### GIT Hub Tags:
#A11yWCAG2.2; #A11yTCS; #A11ySev3; #Testing, Interactive and ARM tools; #BM_Testing-Interactive and ARM tools_Win32_JULY2024; #DesktopApp; #WCAG2.5.7; #FTP; #A11yMAS; #Dragging Movements;

### Environment and OS details:
Windows 11 enterprise
OS build 22631.3737

### Reproduction Steps

1. Install "Visual Studio Code" application.
2. Install .NET 6.0 SDK using https://dotnet.microsoft.com/en-us/download/dotnet/thank-you/sdk-6.0.402-windows-x64-installer.
3. Install Polyglot Notebook extension.
4. launch Visual studio code insider and Ctrl+Alt+Shift+N will invoke "Polyglot Notebook: Create new blank notebook" command used above.
5. Enter any code and run the code.
6. Tab till values and press enter.
7. In polyglotntebooks:values section tab till "table".
8. Observe that No single pointer provided to drag the "columns headers" to the required position in the "Polyglot Notebook" grid table.

### Actual behavior:
No single pointer provided to drag the "columns headers" to the required position in the "Polyglot Notebook" grid table.

### Expected behavior:
Single pointer provided to drag the "columns headers" to the required position in the "Polyglot Notebook" grid table.

### User Impact:
People with motor impairments who cannot operate content because the only way to actuate a function is by dragging a target element from its initial position to some other position.

### Attachments:
[No single pointer provided to drag the columns headers to the required position in the Polyglot Notebook grid table.webm](https://github.com/dotnet/interactive/assets/98728920/bf4bc39f-c274-459e-8dcc-2032f1d94a42)
